app/decorators/user_decorator.rb

```diff
@@ -2,6 +2,10 @@
 
 class UserDecorator < SimpleDelegator
 
+  class << self
```

Comment:
because the best_in_place gem did a funny thing when checking the class of the attribute you're editing